app/lib/shared/configuration.dart

```diff
@@ -255,11 +269,9 @@ class Configuration {
   factory Configuration.fromEnv(EnvConfig env) {
     if (env.gcloudProject == 'dartlang-pub') {
       return Configuration._prod();
-    } else if (env.gcloudProject == 'dartlang-pub-dev') {
-      return Configuration._dev();
-    } else {
-      throw Exception('Unknown project id: ${env.gcloudProject}');
-    }
+    } else if (env.gcloudProject.startsWith('dartlang-pub-dev')) {
```

Comment:
Let's remove this for now, we don't want to establish such pattern - it could lead to weird name collisions and name squatting.